### PR TITLE
Set lib target framework to netstandard2.0

### LINKED ISSUE #53: Support netstandard2.0 as a target framework
- OpenAPI.NET targets netstandard2.0.
- We are not able to add OpenAPI.APIManifest as a reference to SemanticKernel as the .NET projects target netstandard2.0 there.

### PR DESCRIPTION
**Pull request by peombwa**
This PR fixes #53 by:
- Changes lib target framework to `netstandard2.0` to be compatible with OpenAPI.NET and Semantic Kernel.
- Sets C# language version to `10` to be compatible with .NET 6, which is supported by Semantic Kernel.
- Addresses changes attributed to the above changes.

### CODE REVIEW

File: src/lib/apimanifest.csproj [hunk 1]
```diff
@@ -30,6 +31,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.OpenApi.Readers" Version="1.6.11" />
+    <PackageReference Include="System.Text.Json" Version="8.0.0" />
```
Comment (baywet) [inline src/lib/apimanifest.csproj:34]: this should be a bracket dependency so it works on lower runtimes

**Review by baywet — CHANGES_REQUESTED** [review]
I get why we're doing that, I'm just not happy to see the code go back in time 💩💩💩💩

File: src/lib/apimanifest.csproj (hunk 1 above)
Comment (baywet) [reply src/lib/apimanifest.csproj]: 5 might bring a broader compatibility with non Microsoft runtimes. Even though the use case might be very narrow.

**Review by zengin — APPROVED** [review]

File: src/lib/apimanifest.csproj (hunk 1 above)
Comment (peombwa) [reply src/lib/apimanifest.csproj]: v5.x of the lib has been deprecated - https://www.nuget.org/packages/System.Text.Json/5.0.0. v6 seems appropriate for now.

**Review by baywet — APPROVED** [review]

**Pull request merged**